abce/agent.py

```diff
@@ -48,6 +48,111 @@
 import traceback
 from abce.notenoughgoods import NotEnoughGoods
 import datetime
+from abce.trade import get_epsilon
+epsilon = get_epsilon()
+
+
+class Inventory(defaultdict):
+    def __init__(self, name):
+        super(Inventory, self).__init__(float)
+        self.name = name
+        self.contracts = set()
```

Comment:
Make sense, for the logical structure. Unless there are agent actions that can be applied to both goods and contracts.